samples/02_power_users_developers/count_the_usage_frequency_of_api_methods.ipynb

```diff
@@ -0,0 +1,986 @@
+{
```

Comment:
<p>Perhaps you want to explain why we want to differentiate <code>result</code> and <code>result2</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/514/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>